src/language-markdown/utils.js

```diff
@@ -152,11 +151,16 @@ function splitText(text, options) {
     }
     nodes.push(node);
 
-    function isBetween(kind1, kind2) {
+    /**
+     * @param {WordNode} lastNode
+     * @param {any} node
+     * @param {string} kind1
+     * @param {string} kind2
+     * @returns
+     */
+    function isBetween(lastNode, node, kind1, kind2) {
```

Comment:
I really don't know types, and even don't like types. But this is really silly to pass a local variable to the function.

Reply:
I agree with that, but I don't like using ts-expect-error or ts-ignore more than necessary.